components/registry/org.wso2.carbon.registry.extensions/src/main/java/org/wso2/carbon/registry/extensions/handlers/SwaggerMediaTypeHandler.java

```diff
@@ -112,6 +113,22 @@ public void setEndpointLocationConfiguration(OMElement locationConfiguration) {
 	}
 
 	/**
+	 * @return createService
+	 */
+	public boolean isCreateService() {
+		return createService;
+	}
+
+	/**
+	 * Extracts createService property from the registry.xml
+	 *
+	 * @param createService createService property
+	 */
+	public void setCreateService(String createService) {
```

Comment:
@sinthuja This method is invoked through RegistryConfigurationProcessor at the server startup and the argument `createService` is extracted from the `registry.xml` where the handler is registered. This method converts the String value read from the `registry.xml` and keep it as a boolean variable. This boolean value is passed to the `SwaggerProcessor` object through the constructor(Please refer line 166, 173) and used in to manipulate the creation of Rest Service artifact when a swagger resource is added to the registry.